unity/Holo/Assets/Scripts/ModelWithPlate.cs

```diff
@@ -9,12 +9,15 @@ public class ModelWithPlate : MonoBehaviour, IClickHandler
     public GameObject ButtonsModel;
     public GameObject ButtonsModelPreview;
     public GameObject PlateAnimated;
+    public GameObject modelClipPlane;
 
     private GameObject collectionButtons;
 
     private void Start()
     {
         collectionButtons = GameObject.Find("buttonsAdd");
+        modelClipPlane = GameObject.Find("modelClipPlane");
+        modelClipPlane.SetActive(false);
         RefreshUserInterface();

```

Comment:
I would advise to avoid searching for game objects by name. Instead assign the public properties (like `ButtonsAdd` and `ModelClipPlane` defined above) in Unity Editor.

This is consistent with how I exposed `ButtonsModel`  etc. in `ModelWithPlate.cs`.

This solution "scales" a bit better --- you can attach different GameObjects (by dragging in editor, or by code) without the need to rename stuff in scene. It also makes sure that prefabs are self-contained, and do not reference stuff on particular scene.

Reply:
I agree, this is WIP version to check if clipping plane mechanics works.
I cannot atach `mainModel` by dragging from editor cause this model is instantiated from code and does not exists in editor :) But I will drag here the whole display model or other way round - connect this clipping plane to display model script so it can set the model once it is loaded.